src/Microsoft.ML.OnnxTransformer/DnnImageFeaturizerTransform.cs

```diff
@@ -41,11 +41,25 @@ internal DnnImageFeaturizerInput(string outputColumnName, string inputColumnName
     }
 
     /// <summary>
-    /// The Dnn Image Featurizer is just a wrapper around two <see cref="OnnxScoringEstimator"/>s and three <see cref="ColumnCopyingEstimator"/>
+    /// The Dnn Image Featurizer is just a wrapper around two <see cref="OnnxScoringEstimator"/> and three <see cref="ColumnCopyingEstimator"/>
     /// with present pretrained DNN models. The ColumnsCopying are there to allow arbitrary column input and output names, as by default
     /// the ONNXTransformer requires the names of the columns to be identical to the names of the ONNX model nodes.
     /// Note that because of this, it only works on Windows machines as that is a constraint of the OnnxTransformer.
```

Comment:
>es  [](start = 66, length = 3)

comma, I believe.

Reply:
Windows machines, (add comma)

---
In reply to: [277200898](https://github.com/dotnet/machinelearning/pull/3456#discussion_r277200898) [](ancestors = 277200898)